tests/scattering2d/test_torch_scattering2d.py

```diff
@@ -302,31 +293,43 @@ def test_device_mismatch(self, backend_device):
 class TestFFT:
     @pytest.mark.parametrize('backend', backends)
     def test_fft(self, backend):
-        x = torch.randn(2, 2, 2)
+        import numpy as np
+        
+        x_np = np.random.rand(4, 4, 2)
 
-        y = torch.empty_like(x)
-        y[0, 0, :] = x[0, 0, :] + x[0, 1, :] + x[1, 0, :] + x[1, 1, :]
-        y[0, 1, :] = x[0, 0, :] - x[0, 1, :] + x[1, 0, :] - x[1, 1, :]
-        y[1, 0, :] = x[0, 0, :] + x[0, 1, :] - x[1, 0, :] - x[1, 1, :]
-        y[1, 1, :] = x[0, 0, :] - x[0, 1, :] - x[1, 0, :] + x[1, 1, :]
+        x = torch.from_numpy(x_np)
+        
+        x_r_np_fft = np.fft.fft2(x_np[..., 0])
 
```

Comment:
I ended up comparing against the numpy outputs... would we rather have an example that has formulas done by hand?

I was planning on doing that, but I realized the amount of code would be a decent amount. Thought it was easier to do this.

Reply:
I ended up calculating the dft using formulas in numpy. is this fine, or would you like something more specific?

Reply:
I don't quite understand. What would be more specific?

Reply:
Sounds like this is good then!

Reply:
(I was almost worried you wanted me to type out each calculation of the coefficents heh heh)

Reply:
No what I was saying was that you can fix a *single* frequency and type out its formula. That being said, what you have works (although the code could be vectorized to get something more compact and faster).

Reply:
Yeah that makes sense. Think you could name a few functions/gimme some hints that could vectorize this stuff?